prometheus_es_exporter/__init__.py

```diff
@@ -292,6 +292,10 @@ def main():
     parser = argparse.ArgumentParser(description='Export ES query results to Prometheus.')
     parser.add_argument('-e', '--es-cluster', default='localhost',
                         help='addresses of nodes in a Elasticsearch cluster to run queries on. Nodes should be separated by commas e.g. es1,es2. Ports can be provided if non-standard (9200) e.g. es1:9999 (default: localhost)')
+    parser.add_argument('--client-cert',
+                        help='Certificate for authentication. (default: no cert)')
+    parser.add_argument('--client-key',
+                        help='Key for authentication. (default: no key)')
```

Comment:
It'd be nice to have a bit more explanation for these arguments in the help messages. The `--ca-certs` help message below is a good example.

Reply:
(They should probably be below that argument as well)